packages/jamf_protect/kibana/dashboard/jamf_protect-e9b86210-c65c-11ee-882f-57f79af43d7f.json

```
@@ -347,7 +345,7 @@
                             "internalReferences": [],
                             "query": {
                                 "language": "kuery",
-                                "query": "(data_stream.dataset : \"jamf_protect.alerts\" and event.severity : 3) or (data_stream.dataset : \"jamf_protect.web_threat_events\" and event.severity : 8)"
+                                "query": "event.module : \"jamf_protect\" and event.kind:\"alert\" and (event.severity: 3  or event.severity \u003e= 8)"
```

```suggestion
                                "query": "event.module : \"jamf_protect\" and event.kind:\"alert\" and (event.severity: 3  or event.severity : 8)"
```
(how did that get rendered in?)

---

Honestly i'm not sure - i'm using the `export dashboard` in elastic-package?